build/init/fetch-linux-headers.sh

```diff
@@ -49,7 +49,7 @@ fetch_generic_linux_sources()
 
   echo "Fetching upstream kernel sources for ${kernel_version}."
   mkdir -p "${BUILD_DIR}"
-  curl -sL "https://www.kernel.org/pub/linux/kernel/v${major_version}.x/linux-$kernel_version.tar.gz" \
+  curl -sSL "https://www.kernel.org/pub/linux/kernel/v${major_version}.x/linux-$kernel_version.tar.gz" \
```

Comment:
I'm not sure why `-S` would be needed, and I'm not sure if the errors would be piped to `tar` or would be emitted to stderr without being piped etc. I think though, it makes sense to have `-S` because if there is an error it probably isn't piped to `tar` but instead printed for more verbosity about what went wrong.

```
 -s, --silent             Silent mode
 -S, --show-error         Show error even when -s is used
```